packages/manager/apps/telecom/src/components/notification/notification-list.html

```diff
@@ -103,7 +131,10 @@ <h2 data-translate="components_notification_loading"></h2>
                     data-ng-if="!element.editMode"
                     data-title="{{ 'components_notification_account' | translate }}"
                 >
-                    <span data-ng-bind=":: element.smsAccount"></span>
+                    <span
+                        data-ng-bind=":: element.smsAccount"
+                        data-ng-if="element.type === 'sms' "
```

Comment:
**question:** Can the cell be empty? No placeholder to display then?
Plus there an extra whitespace after `'sms'`.

Reply:
If you add a rule with an e-mail and you have only one SMS account, the SMS account is displayed when it should not be. (if you refresh the page it disappears) The fact of putting this data-ng-if allows to avoid displaying it.